### PR TITLE
feat(lookmovie): add captions support

### PR DESCRIPTION
**Pull request by iMahir**
This pull request resolves #XXX

 - [x] I have read and agreed to the [code of conduct](https://github.com/movie-web/movie-web/blob/dev/.github/CODE_OF_CONDUCT.md).
 - [x] I have read and complied with the [contributing guidelines](https://github.com/movie-web/movie-web/blob/dev/.github/CONTRIBUTING.md).
 - [ ] What I'm implementing was assigned to me and is an [approved issue](https://github.com/movie-web/movie-web/issues?q=is%3Aopen+is%3Aissue+label%3Aapproved). For reference, please take a look at our [GitHub projects](https://github.com/movie-web/movie-web/projects).
 - [x] I have tested all of my changes.
---

Added support for captions in the LookMovie scraper.

### CODE REVIEW

File: src/providers/sources/lookmovie/type.ts [hunk 1]
```diff
@@ -39,8 +39,17 @@ interface VideoSources {
   [key: string]: string;
 }
 
+interface VideoSubtitles {
```
Comment (JipFr) [inline src/providers/sources/lookmovie/type.ts:42]: Do we not already have a captions interface?

Comment (iMahir) [reply src/providers/sources/lookmovie/type.ts:42]: nope

Comment (binaryoverload) [reply src/providers/sources/lookmovie/type.ts:42]: We do which is being used here: https://github.com/movie-web/providers/pull/58/files#diff-cb609b33683c2186160a9afbe487f1e78212bd50d823d357d716564f3ec5d77aR32

This type is the data from Lookmovie I believe

Comment (JipFr) [reply src/providers/sources/lookmovie/type.ts:42]: Yes, we do. There's a `Caption` interface in `@/providers/captions`. Please use that!

Comment (JipFr) [reply src/providers/sources/lookmovie/type.ts:42]: My mistake!

**Review by JipFr — APPROVED** [review]
Looks great to me!

**Review by binaryoverload — APPROVED** [review]

**Pull request merged**